app/routes/planet_routes.py

```diff
@@ -0,0 +1,49 @@
+from flask import Blueprint, jsonify, abort, make_response
+
+class Planet:
+    def __init__(self, id, name, description, has_moon=None):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.has_moon = has_moon
+
+    def make_dict(self):
+        return dict(
+                id = self.id,
+                name = self.name,
+                description = self.description,
+                has_moon = self.has_moon,  
+            )
```

Comment:
👍 

PEP8 (python formatting standard) recommends for named arguments to _not_ but a space around the `=` so that it looks less like we are make variables here. so
```python
    def make_dict(self):
        return dict(
            id=self.id,
            name=self.name,
            description=self.description,
            has_moon=self.has_moon,  
        )
```